javascripts/discourse/initializers/setup.js

```diff
@@ -161,14 +161,14 @@ export default {
     this.keyValueStore = container.lookup("service:key-value-store");
     this.expireOldValues();
 
-    withPluginApi("0.8.7", (api) => {
+    withPluginApi((api) => {
       api.decorateCookedElement(
-        (cooked, postWidget) => {
-          if (!postWidget) {
+        (cooked, helper) => {
+          if (!helper) {
             return;
           }
 
-          const postIdentifier = `${postWidget.widget.attrs.topicId}-${postWidget.widget.attrs.id}-`;
+          const postIdentifier = `${helper.model.topic?.id}-${helper.model.id}-`;
```

Comment:
It means we can get `undefined-1` here, probably not a big deal though as it's not supposed to happen

Reply:
Yep. It's more of a safeguard to prevent a possible crash if by any chance the TC bumps into this scenario.

While we are transitioning between the two rendering systems, the TC can receive a transformed Post or a post model. It doesn't hurt to be extra cautious.